arcgis-ios-sdk-samples/Layers/Raster layer (service)/README.md

```diff
@@ -1,12 +1,32 @@
 # Raster layer (service)
 
-This sample demonstrates how to create and use a raster layer from an ArcGIS Image service.
+Create a raster layer from a raster image service.
 
-![](image1.png)
+![Raster layer (service) sample](raster-layer-service.png)
+
+## Use case
+
+Accessing a raster image from an online service can be useful for analysing the most up-to-date data available for an area. For example, retrieving recent results of bathymetry surveys within a shipping channel monitored for its sediment build-up would allow planners to assess dredging needs.
```

Comment:
```suggestion
Accessing a raster image from an online service can be useful for analyzing the most up-to-date data available for an area. For example, retrieving recent results of bathymetry surveys within a shipping channel monitored for its sediment build-up would allow planners to assess dredging needs.
```
Auto-correction highlighted this one. Omit if it is not the case.